styles/shared/reset.css

```diff
@@ -0,0 +1,26 @@
+/******************************** Resetting Default Styles ************************************************/
+
+body, h1, h2, h3, h4, h5, p, figure, blockquote, dl, dd {
+    margin: 0;
+}
+  
+body {
+    width: 100vw;
+    height: 100vh;
+}
```

Comment:
it is already 100vw 100vh;

Reply:
No, body is not already 100vw and 100vh. Although the use is still debatable I would say min-height:100vh is a better candidate to use but it all depends on the idea of design. @Hnh013 I leave it up to you if you feel need you may take action otherwise I don't think anything is needed here.
we may have a discussion over this if you want @Hnh013 , @shrey27  let me know about it.